src/functionalTest/groovy/com/github/spotbugs/snom/StandardFunctionalTest.groovy

```diff
@@ -251,6 +251,44 @@ spotbugs {
 
         then:
         result.task(':spotbugsMain').outcome == TaskOutcome.SUCCESS
+        result.output.contains('\tat ')
+
+        where:
+        isWorkerApi << [true, false]
+    }
+
+    @Unroll
+    def 'build does not show stack traces when bugs are found with `showStacktraces = true` (Worker API? #isWorkerApi)'() {
```

Comment:
The comment here doesn't match your build file on line 273. Either update the comment or change the test to match the comment.

Reply:
Should be fixed in 7947270. Do you want a squashed single commit?

Reply:
Apologies, that should be d14465d